src/EditorFeatures/Core/Implementation/CodeFixes/CodeFixService.cs

```diff
@@ -304,7 +309,9 @@ public async Task<Document> ApplyCodeFixesForSpecificDiagnosticIdAsync(Document
                     hasFix: d => this.GetFixableDiagnosticIds(fixer, extensionManager).Contains(d.Id),
                     getFixes: dxs =>
                     {
-                        using (RoslynEventSource.LogInformationalBlock(FunctionId.CodeFixes_GetCodeFixesAsync, fixer, cancellationToken))
+                        var fixerName = fixer.GetType().Name;
```

Comment:
Is Name something we want to be showing to users? Also any concern if the name is going to be ambiguious if lots of poeple call it "CodeFixer" or the like?

Reply:
Is there anything else that we can show to the user? I first tried the full type name, but that seemed to verbose. I think just the analyzer/fixer name should be good enough for most practical purposes. We can improve on this if we identify this is not true.